app/views/forms/_expense_form.html.erb

```diff
@@ -1,24 +1,26 @@
+<%= render "shared/forms-list" %>
+<% counter = 0 %>
```

Comment:
why we are using counter here?

Reply:
For passing the hash to hello.js controller when we want to add new row during new submission